ansys/dpf/core/server_factory.py

```diff
@@ -6,7 +6,10 @@
 protocols and server configurations available.
```

Comment:
We need to think about a way of not starting many servers while being in process (maybe it could be managed in that feature)